### PR TITLE
setting up micro-frontends

### PR DESCRIPTION
**Pull request by ScriptedAlchemy**
some changes to how everything works too

### CODE REVIEW

File: src/babel/index.js [hunk 1]
```diff
@@ -272,7 +262,7 @@ module.exports = function dynamicUrlImportPlugin(babel) {
                   return t.assignmentExpression(
                     '=',
                     t.identifier(`const ${moduleName}`),
-                    t.identifier(`__webpack_require__("${moduleName}").default`),
+                    t.identifier(`'lol'`),
```
Comment (ethersage) [inline src/babel/index.js]: lol

Comment (ScriptedAlchemy) [reply src/babel/index.js]: changed it back

File: manual/webpack/webpack.dev.js [hunk 2]
```diff
@@ -6,14 +6,10 @@ const external = require.resolve('../../dist/scout')
...
 
-console.log(URLImportPlugin)
 module.exports = {
   mode: 'development',
   devtool: 'source-map',
-  entry: {
-    main: require.resolve('../src/index.jsx'),
-    other: require.resolve('../src/anoterEntry.js'),
-  },
+  entry: commonPaths[process.env.MFE],
```
Comment (ethersage) [inline manual/webpack/webpack.dev.js]: This is `undefined`

File: manual/webpack/webpack.common.js [hunk 3]
```diff
@@ -3,7 +3,6 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const commonPaths = require('./paths');
 
 module.exports = {
-  entry: commonPaths.entryPath,
```
Comment (ethersage) [inline manual/webpack/webpack.common.js]: `undefined`

File: manual/webpack/webpack.dev.js (hunk 2 above)
Comment (ScriptedAlchemy) [reply manual/webpack/webpack.dev.js]: ahh hold on, this should come from package json

File: manual/webpack/webpack.common.js (hunk 3 above)
Comment (ScriptedAlchemy) [reply manual/webpack/webpack.common.js]: in package json, ```    "manual:websiteOne": "cd Website1; cross-env NODE_ENV=development MFE=1 webpack-dev-server",
    "manual:websiteTwo": "cd Website2; cross-env NODE_ENV=development MFE=2 webpack-dev-server",```

**Comment by ethersage** [comment]
Awesome stuff!

**Comment by ScriptedAlchemy** [comment]
npm published as `0.0.1-beta.6 `

**Pull request merged**